### ISSUE TITLE
Error doing job: Task exception was never retrieved

### ISSUE THREAD

**Issue by gvssr** [postlund/pyatv#1450]
### Describe the bug

Following error shows up a lot in the log since the last few beta releases: `AttributeError: module 'homeassistant.components.zeroconf' has no attribute 'ATTR_TYPE'`

### Error log

```log
Logger: homeassistant
Source: custom_components/apple_tv/config_flow.py:153
Integration: Apple TV
First occurred: November 4, 2021, 20:25:52 (47 occurrences)
Last logged: 10:26:09

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/discovery_flow.py", line 74, in _async_process_pending_flows
    await gather_with_concurrency(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 170, in gather_with_concurrency
    return await gather(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 168, in sem_task
    return await task
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 197, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 224, in _async_init
    result = await self._async_handle_step(flow, flow.init_step, data, init_done)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 320, in _async_handle_step
    result: FlowResult = await getattr(flow, method)(user_input)
  File "/config/custom_components/apple_tv/config_flow.py", line 153, in async_step_zeroconf
    service_type = discovery_info[zeroconf.ATTR_TYPE][:-1]  # Remove leading .
AttributeError: module 'homeassistant.components.zeroconf' has no attribute 'ATTR_TYPE'
```

### How to reproduce the bug?

Honestly not sure, just started showing up after a couple of releases. Noticed the Apple TV also had to be setup manually after hass-atv-beta 1.9.0, instead of being discovered like it used to.

### What is expected behavior?

No errors

### Operating System

core-2021.11.0

### Python

3.9

### pyatv

0.9.5

### Device

Apple TV 4K iOS 15

### Additional context

Integration does seem to work fine.

**Comment by luckystarwy** [comment]
My logs also report the same error

**Comment by postlund** [comment]
Should be fixed in 1.9.1?

**Comment by gvssr** [comment]
Yes, I haven't seen it since 1.9.1, closing this one.
